### DISCUSSION TITLE
HomeKit Adaptive Lighting

### DISCUSSION THREAD

**Discussion by gashead76** [milo526/homebridge-tuya-web#332] [Ideas]
Would it be possible to add the Adaptive Lighting feature to the Tuya Web plugin now that it's supported by Homebridge?

**Comment by milo526** [comment]
I'd really like to get this done, I have already worked on this before, but life is crazy busy right now.
I try to fix the occasional bug but finding time to work on developing new features is near impossible for now (I expect this to be the case until the start of the new year).
If somebody wishes to develop this feature, hit me up and we can see what we can get done!
